src/store/wallet.ts

```diff
@@ -544,7 +543,7 @@ export const actions = actionTree(
     logout({ state, commit }): void {
       state.onboard?.walletReset();
       clearTimeout(getTransactionHistoryAgain);
-      walletData.set({ syncProvider: undefined, syncWallet: undefined, accountState: undefined });
+      walletData.set({ syncWallet: undefined, accountState: undefined });
```

Comment:
Better to use separate method for nulling any internal param / property.
It allows u to keep code cleaner, assign strict types for the methods & protects the data structure.

Reply:
I don't get the point

Reply:
I'm pointing to the separate method on behalf of walletData: e.g. ```walletData.clear()``` which can be called without the params.